ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -122,6 +125,11 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 			return;
 		}
 
+		if (!(await queueService.isQueueStarted())) {
```

Comment:
should this logic be inside of `queueService`? what happens if `queueService.queueWork` is called from somewhere else without calling `registerWorks`? 😬

Reply:
hmmmm, good question